packages/pigeon/platform_tests/alternate_language_test_plugin/example/android/build.gradle

```diff
@@ -16,16 +16,13 @@ allprojects {
         google()
         mavenCentral()
     }
-    gradle.projectsEvaluated {
-        tasks.withType(JavaCompile) {
-            options.compilerArgs << "-Xlint:unchecked" << "-Xlint:deprecation"
-        }
-    }
 }
 
 rootProject.buildDir = '../build'
 subprojects {
     project.buildDir = "${rootProject.buildDir}/${project.name}"
+}
+subprojects {
```

Comment:
🤷. This isn't really a change, it's an older `flutter create` file from an existing project being deleted with the rest of that project, a new `flutter create` file being added as part of the new project, and git noticing that they are pretty close and showing it as a move+edit because rename detection in git is purely similarity based.